docs/user_needs/zephyr_rtos_user.sdoc

```diff
@@ -0,0 +1,87 @@
+[DOCUMENT]
+TITLE: Zephyr RTOS User
+REQ_PREFIX: ZEP-
+
+[GRAMMAR]
+IMPORT_FROM_FILE: user_needs.sgra
+
+[FREETEXT]
+
+TBD
+
+[/FREETEXT]
+
+[USER_NEED]
+UID: ZEP-127
+STATUS: Draft
+TYPE: User Need
+USER: Product Manufacturer
+TITLE: OSS License Allows Statically Linked Private Extensions
+STATEMENT: >>>
+As a Product Manufacturer of resource-constrained devices,
+
+I need open-source software to be licensed such that the proprietary code statically linked with the open source software can be kept private.
+<<<
+
+[USER_NEED]
+STATUS: Draft
+TYPE: User Need
+USER: Zephyr RTOS User
+TITLE: Easily Switch Application to different MCU Architecture
+STATEMENT: >>>
+As a Zephyr RTOS user,
+
+I want to be able to easily switch my application to a different MCU architecture (x86, ARM Cortex-M/A, RISCV etc.).
+<<<
```

Comment:
1) RTOS user is ambiguous. Is it a software developer or a user of Zephyr RTOS in the context of a larger project?

2) How to quantify the word "easily"? E.g., "predictably low effort"?

3) Should the wording include "portable" or "Portability"?